### PR TITLE
[admin]create question+answer

### PR DESCRIPTION
**Pull request by dat-hedspi**

### CODE REVIEW

**Comment by tuannd255** [comment]
ok

File: app/views/admin/questions/_answer_fields.html.erb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<fieldset class="field">
+  <%= f.label :answer %>
+  <%= f.text_field :answer, class: "form-control" %>
+  <div class="correct-choose">
+    <%= f.label :is_correct %>
+    <%= f.check_box :is_correct, class: "check" %>
+  </div>
+  <div class="remove-choose">
+    <%= f.hidden_field :_destroy %>
+    <%= link_to t("question.remove"), "#", class:"remove_fields" %>
```
Comment (tuannd255) [inline app/views/admin/questions/_answer_fields.html.erb]: ident

**Comment by tuannd255** [comment]
ok

**Comment by hatd** [comment]
ok

**Pull request closed without merging**